plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -281,6 +296,18 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
         }
       }
 
+      if (!isActionAuthorized(decision, { action: action.id, input })) {
+        throw new InputError(
```

Comment:
Do we have a better error here than `InputError`?

Reply:
Good point, switched to `NotAllowedError`